File: curso python/semestre/2areaperimetroCIRCULO.py
#calculo del area y perimetro de un circulo conocido su diametro;
import math
print ("calculo de AREA Y PERIMETRO de un CIRCULO")
diametro = int (input("introduce el diametro del circulo: "))
radio = diametro/2
area= math.pi * pow (radio, 2)
perimetro= 2* math.pi* radio
print ("el area del circulo es: ", area)
print ("el perimetro del circulo es: ", perimetro)
#otra forma
print ("OTRA FORMA CON FUCNION")

def area():
    diametro= float(input("introduce el diamtro del circulo: "))
    radio = diametro / 2
    areaa = math.pi * pow(radio,2)
    return areaa

resultado1 = area()
print ("primer area es: ", resultado1)
resultado2= area()
print ("segundo area es: ", resultado2)

File: curso c++/semestre/2areaPERIMETROcirculo.cpp
#include <iostream>
#include <cmath> 
using namespace std;
int main (){

    //calular el are  y perimetr de un circulo conocido su diametro;
    //variables
    double area,perimetro, diametro,radio;
    //entrada
    cout << "inserte el diametro  del circulo: ";
    cin >> diametro;
    //proceso
    radio = diametro/2;
    area = M_PI * pow(radio,2);
    perimetro =3.1416 * diametro;
    //salida
    cout << "area: "<<area<<endl;
    cout << "perimetro: "<< perimetro;


    return 0;

}